<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemEvaluationWindow</class>
 <widget class="QMainWindow" name="SystemEvaluationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>System Evaluation</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res/icons.qrc">
    <normaloff>:/icons/MainIcon.svg</normaloff>:/icons/MainIcon.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" rowspan="2">
       <widget class="DRMDetail" name="drmDetail" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="GroupBoxGPSStatus">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>GPS</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="TextLabelGPSPosition">
           <property name="text">
            <string>Lat: 53.123°   Long: 00.456°</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="TextLabelGPSTime">
           <property name="text">
            <string>24/01/2007 15:14.33 UTC</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="TextLabelGPSSpeedHeading">
           <property name="text">
            <string>10.0 m/s  151°</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="CMultColorLED" name="LEDGPS" native="true">
           <property name="minimumSize">
            <size>
             <width>13</width>
             <height>13</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>13</width>
             <height>13</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="SDCdateTime">
        <property name="text">
         <string>time</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTreeWidget" name="chartSelector">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="rootIsDecorated">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string>Chart Selector</string>
         </property>
        </column>
        <item>
         <property name="text">
          <string>Spectrum</string>
         </property>
         <property name="flags">
          <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
         <item>
          <property name="text">
           <string>Input PSD</string>
          </property>
          <property name="icon">
           <iconset resource="res/icons.qrc">
            <normaloff>:/icons/CSSpectrumInpPSD.png</normaloff>:/icons/CSSpectrumInpPSD.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Input Spectrum</string>
          </property>
          <property name="icon">
           <iconset resource="res/icons.qrc">
            <normaloff>:/icons/CSSpectrumInpSpectr.png</normaloff>:/icons/CSSpectrumInpSpectr.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Waterfall Input Spectrum</string>
          </property>
          <property name="icon">
           <iconset resource="res/icons.qrc">
            <normaloff>:/icons/CSSpectrumWaterf.png</normaloff>:/icons/CSSpectrumWaterf.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Shifted PSD</string>
          </property>
          <property name="icon">
           <iconset resource="res/icons.qrc">
            <normaloff>:/icons/CSSpectrumShiftedPSD.png</normaloff>:/icons/CSSpectrumShiftedPSD.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Audio Spectrum</string>
          </property>
          <property name="icon">
           <iconset resource="res/icons.qrc">
            <normaloff>:/icons/CSSpectrumAudio.png</normaloff>:/icons/CSSpectrumAudio.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>SNR Spectrum</string>
          </property>
          <property name="icon">
           <iconset resource="res/icons.qrc">
            <normaloff>:/icons/CSSpectrumSNR.png</normaloff>:/icons/CSSpectrumSNR.png</iconset>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>Channel</string>
         </property>
         <property name="flags">
          <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
         <item>
          <property name="text">
           <string>Impulse Response</string>
          </property>
          <property name="icon">
           <iconset resource="res/icons.qrc">
            <normaloff>:/icons/CSChannel.png</normaloff>:/icons/CSChannel.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Transfer Function</string>
          </property>
          <property name="icon">
           <iconset resource="res/icons.qrc">
            <normaloff>:/icons/CSTF.png</normaloff>:/icons/CSTF.png</iconset>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>Constellation</string>
         </property>
         <property name="flags">
          <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
         <item>
          <property name="text">
           <string>FAC / SDC / MSC</string>
          </property>
          <property name="icon">
           <iconset resource="res/icons.qrc">
            <normaloff>:/icons/CSConstellation.png</normaloff>:/icons/CSConstellation.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>FAC</string>
          </property>
          <property name="icon">
           <iconset resource="res/icons.qrc">
            <normaloff>:/icons/CSFAC.png</normaloff>:/icons/CSFAC.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>SDC</string>
          </property>
          <property name="icon">
           <iconset resource="res/icons.qrc">
            <normaloff>:/icons/CSSDC.png</normaloff>:/icons/CSSDC.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MSC</string>
          </property>
          <property name="icon">
           <iconset resource="res/icons.qrc">
            <normaloff>:/icons/CSMSC.png</normaloff>:/icons/CSMSC.png</iconset>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>History</string>
         </property>
         <property name="flags">
          <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
         <item>
          <property name="text">
           <string>SNR / Audio</string>
          </property>
          <property name="icon">
           <iconset resource="res/icons.qrc">
            <normaloff>:/icons/CSAudSNRHist.png</normaloff>:/icons/CSAudSNRHist.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Delay / Doppler</string>
          </property>
          <property name="icon">
           <iconset resource="res/icons.qrc">
            <normaloff>:/icons/CSHistoryDelDopp.png</normaloff>:/icons/CSHistoryDelDopp.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Frequency / Sample Rate</string>
          </property>
          <property name="icon">
           <iconset resource="res/icons.qrc">
            <normaloff>:/icons/CSHistoryFreqSam.png</normaloff>:/icons/CSHistoryFreqSam.png</iconset>
          </property>
         </item>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QwtPlot" name="plot">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="lineWidth">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="DRMOptions" name="drmOptions" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>10</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="GroupBoxLogFileSaveData">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Audio</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QCheckBox" name="CheckBoxMuteAudio">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Mute &amp;Audio</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="CheckBoxReverb">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>&amp;Reverberation</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Recording</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QCheckBox" name="CheckBoxWriteLog">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>&amp;Log to file</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="CheckBoxSaveAudioWave">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>&amp;Save Audio</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="_35">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="Spacer3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="buttonOk">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="windowTitle">
             <string/>
            </property>
            <property name="text">
             <string>&amp;Close</string>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
            <property name="default">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CMultColorLED</class>
   <extends>QWidget</extends>
   <header location="global">MultColorLED.h</header>
  </customwidget>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DRMDetail</class>
   <extends>QWidget</extends>
   <header>drmdetail.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DRMOptions</class>
   <extends>QWidget</extends>
   <header>drmoptions.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>SystemEvaluationWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>723</x>
     <y>605</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
